packages/core/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "rpaframework-core"
-version = "6.5.1"
+version = "6.5.2"
```

Comment:
Since we're upgrading the core version here, after merge and release, we should upgrade to `rpaframework-core = "^6.5.2"` dependency in **rpaframework-windows** as well, right?